Gems/ROS2/Code/Source/RobotImporter/Utils/SourceAssetsStorage.cpp

```diff
@@ -78,7 +79,19 @@ namespace ROS2::Utils
                 t.m_assetId = info.m_assetId;
                 t.m_sourceAssetGlobalPath = fullSourcePathStr;
                 t.m_productAssetRelativePath = info.m_relativePath;
-                availableAssets[crc] = t;
+                if (availableAssets.contains(crc))
```

Comment:
suggestion: This is pretty minor (feel free to ignore) but calling `contains` on the map and then using the `[]` operator is technically doing more work than is needed, we could instead use `find` and then write to the iterator if it's not equal to `end()`. Something like...

```c++
auto availableAssetIt = availableAssets.find(crc);
if (availableAssetIt != availableAssets.end()) {
    ...
    *it = availableAsset;
} else {
    availableAssets.insert({crc, availableAsset});
}
```